### ISSUE TITLE
[indent] - Feature request - Allow HTML, HEAD and BODY to be on same level

### ISSUE THREAD

**Issue by gian1200** [yeonjuan/html-eslint#101]
Is it possible to add a configuration to optionally allow this scenario?

Some of the projects where I'm advocting to implement "@html-eslint" use this format.

```html
<!DOCTYPE html>
<html lang="es">

<head>
	...
</head>

<body>
	...
</body>

</html>
```

Current message:
- Expected indentation of 2 tab but found 1 tab. `eslint(@html-eslint/indent)`

**Comment by yeonjuan** [comment]
Hi @gian1200 :) thanks for the suggestion. I think we can add options on @html-eslint/indent.

**Comment by gian1200** [comment]
Now that indent for template literals is being modified (https://github.com/yeonjuan/html-eslint/issues/241), could it be possible to include also the indent feature from this issue?

**Comment by yeonjuan** [comment]
@gian1200 Yes that sounds good, I think we should add a new option for that.

**Comment by yeonjuan** [comment]
@gian1200 I'm thinking of the following options What do you think?
```json
{
  "@html-eslint/indent": [
    "error",
    2,
    {
      "Tag": {
           "html": 0 // If the tag name is html, increment indent by 0.
       }
    }
  ]
}
```

**Comment by gian1200** [comment]
Interesting approach. That way the option can be used for any tag (not only HTML/BODY/HEAD), correct?

Your example is something like "_for HTML tag, all it's children will have 0 indentation, relative to current HTML indentation_".

Looks good to me. Maybe just not use capital letter for "Tag", or change it to a more clear name like "childrenIndent"/"childrenindentation" or any other one you consider more appropriate

**Comment by gian1200** [comment]
PS: I use tabs, but I guess this will also work with it

**Comment by yeonjuan** [comment]
@gian1200  Okay, I think `tagChildrenIndent` would be more clear. I'll create a PR, thank you!

**Comment by yeonjuan** [comment]
It's released in `0.32.0`. Thanks

**Comment by gian1200** [comment]
Awesome! Happy new year! Will try it right away!
